### ISSUE TITLE
Tracking further kinds of PRs

### ISSUE THREAD

**Issue by grunweg** [jcommelin/queueboard#11]
What  do you think about including a few more categories:
1. PRs looking for an owner: labelled with please-adopt or help-wanted

2. PRs stalled on a decision: not sure how to define this; awaiting-zulip and no activity in a while could be one way. (I don't think we're good tagging such PRs currently --- but if these are tracked specially, that could change. If *I* had a way to surface "this is blocked on a decision" PRs more prominently, I would certainly use it more.)

3. PRs awaiting merge: all PRs which are just blocked on a merge --- CI passes, not WIP or blocked, but a merge conflict 

4. I think it could also be interesting to have a list of "PRs falling through the cracks", whose status we cannot determine otherwise. A first approximation could be [this list](https://github.com/leanprover-community/mathlib4/pulls?q=is%3Aopen+is%3Apr+-label%3Adelegated+-label%3Aawaiting-author+-label%3AWIP+-label%3Ablocked-by-other-PR+-is%3Adraft+-label%3Amerge-conflict+status%3Afailure) of PRs which
  - are not blocked on another PR
  - not awaiting-author, awaiting-zulip or awaiting-CI
  - are not marked WIP or draft
  - are not ready-to-merge or delegated
  - fail CI (merge conflicts can be tracked separately)

**Comment by grunweg** [comment]
#17 implements items (1) and (3). The others would not be difficult either, but let's wait until this board is actually used to decide so.

**Comment by grunweg** [comment]
#39 implements item (2), by simply following the "needs-decision" label

**Comment by grunweg** [comment]
In light of the new statistics section (and automatic classification of PRs according to their state), item (4) is essentially obsolete.
The only step possibly worth doing is showing *which* PRs fit into a certain category: this should be easy to do, and can be done if there is a need later.
